prime-angular-frontend/src/app/shared/pipes/certificate.pipe.ts

```diff
@@ -21,9 +21,8 @@ export class CertificatePipe implements PipeTransform {
 
   private getFullName(certificate: any) {
```

Comment:
Note for another future ticket so that one of us remembers, but we have I believe 3 pipes that do full name